src/sec_related_rates.mbx

```diff
@@ -269,232 +269,303 @@
+              $a = 0.0573;
+              $b = 0.0725;
+              $c = 0.0733;
             </pg-code>
           </setup>
           <statement>
+<!--FUNGUS: With the picture, this question seems okay, but without it, the aircraft's minimum distance is  10,000 feet away. The ground distance should be referenced, for maximum clarity.-->
+<!--Except, there's no code here to embed the picture like there is for the question with the ladder. But there is a picture in the printed book I have.-->
+<!--FUNGUS: This problem makes no sense. You have to lead a target to actually hit it. Are we assuming the gun rotates to a spot in front of the aircraft that accounts for travel time of aircraft, projectile, and detonation? Real-world problems should be actually real. Make it a camera tracking a plane or something instead.-->
+<!--I feel like this question should have prompting regarding the appropriate units to be used in the answer. Radians/second isn't something students are likely to be familiar with.-->
             <p>An F-22 aircraft is flying at 500mph with an elevation of 10,000ft on a straight<ndash />line path that will take it directly over an anti<ndash />aircraft gun.</p>
-
-            <!-- START\1END -->
-
             <p>How fast must the gun be able to turn to accurately track the aircraft when the plane is:</p>
 
             <ol>
-              <li><p>1 mile away?</p></li>
-
-              <li><p>1/5 mile away?</p></li>
-
-              <li><p>Directly overhead?</p></li>
+              <li><p>1 mile away?</p>
+                <p><var name="$a" width="10"/></p>
+              </li>
+              <li><p>1/5 mile away?</p>
+                <p><var name="$b" width="10"/></p>
+              </li>
+              <li><p>Directly overhead?</p>
+                <p><var name="$c" width="10"/></p>
+              </li>
             </ol>
           </statement>
-          <solution>
-            <p>Due to the height of the plane, the gun does not have to rotate very fast.</p>
-
-            <ol>
-              <li><p><m>0.0573</m> rad/s</p></li>
-
-              <li><p><m>0.0725</m> rad/s</p></li>
-
-              <li><p>In the limit, rate goes to <m>0.0733</m> rad/s</p></li>
-            </ol>
-          </solution>
         </webwork>
       </exercise>
+<!--Exercise 8: Anti-aircraft gun 2-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$a">
+              <static>0.073</static>
+            </var>
+            <var name="$b">
+              <static>3.66</static>
+            </var>
+            <var name="$c">
+              <static>7.33</static>
             </var>
             <pg-code>
+              $a = 0.073;
+              $b = 3.66;
+              $c = 7.33;
             </pg-code>
           </setup>
           <statement>
+<!--FUNGUS: The ensuing class discussion regarding this pilot's death wish will certainly be lively-->
```

Comment:
The author teaches at VMI. Maybe this influences the military setting of some examples?